### PR TITLE
Feature: Added tooltips to Layout Picker

### PR DESCRIPTION
**Pull request by 0x5bfa**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   For #10492

## Summary

- Added tooltip converter for Slider
- Added xml comments

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Go to a folder
   2. Open Layout flyout
   3. See the tooltip is shown on hover

## Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Converters/EnumValueToEnumNameConverter.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+	internal sealed class EnumValueToEnumNameConverter : IValueConverter
+	{
+		public string EnumTypeName { get; set; } = string.Empty;
+
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			var stringValue = value.ToString() ?? string.Empty;
+
+			return EnumTypeName switch
```
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/Converters/EnumValueToEnumNameConverter.cs:16]: Oh, in that case we should create dictionaries like others in Settings dialog and [Compatibility page (still PR)](https://github.com/files-community/Files/pull/14473). Should create them in factories namespace.

File: src/Files.App/Data/Factories/LocalizedEnumDescriptionFactory.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+				GridViewSizeKinds.Add(GridViewSizeKind.Three, "Three".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Four, "Four".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Five, "Five".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Six, "Six".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Seven, "Seven".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Large, "Large".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Nine, "Nine".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Ten, "Ten".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.Eleven, "Eleven".GetLocalizedResource());
+				GridViewSizeKinds.Add(GridViewSizeKind.ExtraLarge, "ExtraLarge".GetLocalizedResource());
```
Comment (yaira2) [inline src/Files.App/Data/Factories/LocalizedEnumDescriptionFactory.cs:70]: Can you match these up with the strings in the resource file?

Comment (0x5bfa) [reply src/Files.App/Data/Factories/LocalizedEnumDescriptionFactory.cs]: Yep. Did.

**Comment by 0x5bfa** [comment]
I think this is now ready.

**Comment by yaira2** [comment]
> I've added Tiles as well. Should remove because there's one option and slider is not available?

We might as well keep it

File: src/Files.Core/Data/Enums/GridViewSizeKind.cs [hunk 3]
```diff
@@ -3,19 +3,69 @@
 
 namespace Files.Core.Data.Enums
 {
+	/// <summary>
+	/// Defines constant that specifies the icon/height size in Grid layout.
```
Comment (yaira2) [inline src/Files.Core/Data/Enums/GridViewSizeKind.cs:7]: This isn't specific to the icon height

**Review by yaira2 — DISMISSED** [review]
LGTM

**Comment by yaira2** [comment]
>  Did you check for accessibility? You can use Accessibility Insights for this.

@0x5bfa does the screen reader work correctly?

File: src/Files.Core/Data/Enums/GridViewSizeKind.cs (hunk 3 above)
Comment (yaira2) [reply src/Files.Core/Data/Enums/GridViewSizeKind.cs]: I updated the comments accordingly

**Pull request merged**